Document/MediaManager.php

```diff
@@ -34,7 +34,7 @@ public function save($entity, $andFlush = true)
             $entity->setProviderName(func_get_arg(2));
         }
 
-        if ($andFlush && is_bool($andFlush)) {
+        if (is_bool($andFlush)) {
```

Comment:
How about you checkout this branch, run tests, checkout HEAD^, run tests again.
I've separated test and code commits so you can easily find out how much it actually helps.
Also read test code to understand why one might want to pass false there